easypy/threadtree.py

```diff
@@ -168,27 +177,77 @@ def iter_thread_frames():
         yield from sys._current_frames().items()
 
 
+class FrameWrapper:
```

Comment:
Let's be more meaningul:

```suggestion
class FrameWithThreadIdent:
```